obal/data/modules/tito_build.py

```diff
@@ -36,6 +38,13 @@ def main():
         if value:
             command += ['--' + param, value]
 
+    if module.params['builder']:
```

Comment:
You could add `builder` to the tuple on line 36 since this behaves just as `--dist`, `--scl` and `--output`.